obspy/signal/spectral_estimation.py

```diff
@@ -1942,8 +1956,15 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 color = "black"
             ax.plot(xdata, mean_, color=color, zorder=9)
 
+        # Choose the correct noise moel for infrasound
+        if self.special_handling == "infrasound":
+            # Re-name noise model file
+            mod = NOISE_MODEL_FILE_INF
+        else:
+            mod = NOISE_MODEL_FILE
+
         if show_noise_models:
-            for periods, noise_model in (get_nhnm(), get_nlnm()):
+            for periods, noise_model in (get_nhnm(mod), get_nlnm(mod)):
```

Comment:
This could be a bit confusing, since `nhnm` stands for "New High Noise Model" which is a term coined in that Peterson paper. Also having this as an `arg` will break peoples' codes, potentially, or rather likely.
I'd prefer a new helper method to fetch the infrasound models instead.

```

        if self.special_handling == "infrasound":
            # Re-name noise model file
            models = (get_boide_high_noise(), get_boide_low_noise())  # or some other appropriate naming
        else:
            models = (get_nhnm(), get_nlnm())

        if show_noise_models:
            for periods, noise_model in models:
```

Reply:
This is much better - thank you!